<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Window_Registro</class>
 <widget class="QMainWindow" name="Window_Registro">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>536</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>536</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>536</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QVBoxLayout{
	background-color:transparent
}</string>
   </property>
   <widget class="QLabel" name="Label_Fondo">
    <property name="geometry">
     <rect>
      <x>-20</x>
      <y>0</y>
      <width>976</width>
      <height>541</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(39, 125, 161, 200);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../QRC/imagenesInicioSesion.qrc">:/newPrefix/Images/viejito-01.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>60</y>
      <width>321</width>
      <height>411</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetNoConstraint</enum>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="Label_Nombre">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #FFFFFF; background-color: transparent;</string>
       </property>
       <property name="text">
        <string>Nombre</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="Line_Nombre">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:#FFFFFF; color:#01233C; border-radius:10px</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Label_Apellido">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #FFFFFF; background-color: transparent;</string>
       </property>
       <property name="text">
        <string>Apellido</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="Line_Apellido">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:#FFFFFF; color:#01233C; border-radius:10px</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Label_Email">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #FFFFFF; background-color: transparent;</string>
       </property>
       <property name="text">
        <string>Email</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="Line_Email">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:#FFFFFF; color:#01233C; border-radius:10px</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Label_Contrasena">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #FFFFFF; background-color: transparent;</string>
       </property>
       <property name="text">
        <string>Contrase単a</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="Line_Contrasena">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:#FFFFFF; color:#01233C; border-radius:10px</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="placeholderText">
        <string>Contrase単a</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Label_ConfirmarContrasena">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #FFFFFF; background-color: transparent;</string>
       </property>
       <property name="text">
        <string>Confirmar contrase単a</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="Line_ConfirmarContrasena">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:#FFFFFF; color:#01233C; border-radius:10px</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="placeholderText">
        <string>Contrase単a</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Label_Fecha">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #FFFFFF; background-color: transparent;</string>
       </property>
       <property name="text">
        <string>Fecha de nacimiento</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="Fecha">
       <property name="font">
        <font>
         <family>Microsoft YaHei</family>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:#FFFFFF; color:#01233C; border-radius:10px</string>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::UpDownArrows</enum>
       </property>
       <property name="accelerated">
        <bool>false</bool>
       </property>
       <property name="date">
        <date>
         <year>1990</year>
         <month>1</month>
         <day>1</day>
        </date>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>5</width>
         <height>25</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="Boton_Registro">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>8</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	background-color:#FFFFFF; 
	color:#01233C; 
	border-radius:13px;
}

QPushButton:hover{
	background-color:#01233C; 
	color:#FFFFFF; 
	border-radius13:px;
}</string>
       </property>
       <property name="text">
        <string>Registrarse</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="Label_Fonodo_Monocromatico">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>43</y>
      <width>380</width>
      <height>450</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(39, 125, 161, 200);
border-radius : 20px;
opacity: 0.5</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Label_fondo_color">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>536</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#01233C</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>80</y>
      <width>304</width>
      <height>401</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_logo">
       <property name="text">
        <string/>
       </property>
       <property name="textFormat">
        <enum>Qt::PlainText</enum>
       </property>
       <property name="pixmap">
        <pixmap resource="../QRC/imagenesInicioSesion.qrc">:/inicioSesion/Icons/logo.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>27</width>
         <height>49</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="boton_Iniciar_sesion">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
	color: #FFFFFF; 
	background-color: transparent; 
	border-radius:10px;
}
QPushButton:hover{
	color: #00EEFF; 
	background-color: transparent; 
	border-radius:10px;
}
</string>
       </property>
       <property name="text">
        <string>Iniciar Sesion</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <zorder>Label_fondo_color</zorder>
   <zorder>Label_Fondo</zorder>
   <zorder>Label_Fonodo_Monocromatico</zorder>
   <zorder>verticalLayoutWidget</zorder>
   <zorder>verticalLayoutWidget_2</zorder>
  </widget>
 </widget>
 <resources>
  <include location="../QRC/imagenesInicioSesion.qrc"/>
 </resources>
 <connections/>
</ui>
